packages/components/src/tooltip/src/Tooltip.css

```diff
@@ -8,18 +21,104 @@
     overflow-wrap: break-word;
 }
 
-/* We need a .o-ui-tooltip-content class */
+.o-ui-tooltip-arrow,
+.o-ui-tooltip-arrow::before {
+    position: absolute;
+    width: var(--o-ui-tooltip-arrow-dimension);
+    height: var(--o-ui-tooltip-arrow-dimension);
+    z-index: 1;
+}
+
+.o-ui-tooltip-arrow::before {
+    content: "";
+    transform: rotate(45deg);
+    background-color: var(--o-ui-bg-alias-surface);
+    box-shadow: var(--o-ui-bs-alias-floating);
+}
+
+.o-ui-tooltip-arrow::after {
+    content: "";
+    width: 20px;
+    height: 10px;
+    background-color: var(--o-ui-bg-alias-surface);
+    position: absolute;
+    top: calc(var(--o-ui-tooltip-arrow-dimension) / 2);
+    left: calc(-1 * var(--o-ui-tooltip-arrow-dimension) / 2);
+    z-index: 10;
+}
+
 /* LEFT ICON */
-.o-ui-tooltip .o-ui-icon {
+.o-ui-tooltip-wrapper .o-ui-icon {
     margin-right: var(--o-ui-sp-1);
 }
 
 /* RIGHT ICON */
-.o-ui-tooltip .o-ui-text + .o-ui-icon {
+.o-ui-tooltip-wrapper .o-ui-text + .o-ui-icon {
     margin-left: var(--o-ui-sp-1);
 }
 
 /* DISABLED WRAPPER */
 .o-ui-tooltip-disabled-wrapper {
     display: inline-block;
 }
+
+/* POSITION */
+.o-ui-overlay[data-popper-placement^="top"] .o-ui-tooltip-wrapper {
+    padding-bottom: var(--o-ui-tooltip-spacing-with-trigger);
+}
+
+.o-ui-overlay[data-popper-placement^="bottom"] .o-ui-tooltip-wrapper {
+    padding-top: var(--o-ui-tooltip-spacing-with-trigger);
+}
+
+.o-ui-overlay[data-popper-placement^="left"] .o-ui-tooltip-wrapper {
+    padding-right: var(--o-ui-tooltip-spacing-with-trigger);
+}
+
+.o-ui-overlay[data-popper-placement^="right"] .o-ui-tooltip-wrapper {
+    padding-left: var(--o-ui-tooltip-spacing-with-trigger);
+}
+
+/* ARROW  */
+.o-ui-overlay[data-popper-placement^="top"] .o-ui-tooltip-arrow {
+    bottom: calc(var(--o-ui-tooltip-lozenge-dimension) / 2);
+}
+
+.o-ui-overlay[data-popper-placement^="bottom"] .o-ui-tooltip-arrow {
+    top: calc(var(--o-ui-tooltip-lozenge-dimension) / 2);
+}
+
+.o-ui-overlay[data-popper-placement^="left"] .o-ui-tooltip-arrow {
+    right: calc(var(--o-ui-tooltip-lozenge-dimension) / 2);
+}
```

Comment:
those styles used to be located in the Overlay.css. I'm not SUPER happy that i reference o-ui-overlay here, but i don't see any other way. 

We can't trust the "position"/"placement" property set on tooltip, since even if we set "top", if there is no room to display the tooltip at the top, popperjs will show it under, and the data-popper-placement will be updated to bottom, but the property stays at "top"